build/fuseswap-default.tokenlist.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "Fuse Token List",
-  "timestamp": "2021-03-19T20:23:50.267Z",
+  "timestamp": "2021-03-20T18:07:51.462Z",
   "version": {
-    "major": 3,
+    "major": 4,
```

Comment:
why we are bumping the major?

Reply:
changed the address to a checksummed address